tf2onnx/tfonnx.py

```diff
@@ -1803,8 +1802,8 @@ def zeroslike_op(ctx, node, name, args):
 
 _OPSET_6 = {
     "AddN": (direct_op, ["Sum"]),
-    "All": (reduce_logic_op, []),
-    "Any": (reduce_logic_op, []),
+    "All": (reduce_logic_op, ["All", "ReduceMin"]),
```

Comment:
I don't mean this. you can check how reduce_op is reused by Sum/Prod/Min/Mean/Max. Basically it kept original node.